templates/taxi/driver_list.html

```diff
@@ -6,7 +6,9 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          <a href="{% url "taxi:driver-detail" pk=driver.id %}">
```

Comment:
Ensure that `driver.id` corresponds to the correct primary key for the `Driver` model. This is crucial for generating the correct URL for the driver's detail page.